pkgs/development/libraries/rnnoise/default.nix

```diff
@@ -1,16 +1,27 @@
-{ stdenv, lib, fetchFromGitHub, autoreconfHook }:
+{ stdenv, lib, fetchFromGitHub, fetchurl, autoreconfHook }:
 
 stdenv.mkDerivation (rec {
   pname = "rnnoise";
-  version = "2021-01-22";
+  version = "0.2";
 
   src = fetchFromGitHub {
     owner = "xiph";
     repo = "rnnoise";
-    rev = "1cbdbcf1283499bbb2230a6b0f126eb9b236defd";
-    sha256 = "1y0rzgmvy8bf9a431garpm2w177s6ajgf79y5ymw4yb0pik57rwb";
+    rev = "v${version}";
+    hash = "sha256-Qaf+0iOprq7ILRWNRkBjsniByctRa/lFVqiU5ZInF/Q=";
   };
 
+  # Copy from https://raw.githubusercontent.com/xiph/rnnoise/v${version}/model_version
+  model_version = "0b50c45";
```

Comment:
@toastal would something like 134e437adda5a8fb3a324faca9b45af35fb8682a work?

Reply:
@toastal thanks for the feedback but considering there is also a `modelVersion`, I can't put the actual `modelUrl` above in the inputs, since doing so would also allow overriding the `modelVersion` as well. Which should not be, considering they are mutually exclusive option. I think it would be best to implement it this way. https://github.com/NixOS/nixpkgs/pull/305348/commits/6d0941464923786d6405d952ad62bf6023de8c21